other-page/other-page.js

```diff
@@ -1,9 +1,63 @@
-import { checkAuth, logout } from '../fetch-utils.js';
+import { checkAuth, logout, getShoppingList, deleteShoppingList, buyItem, createListItem } from '../fetch-utils.js';
+import { renderItem } from '../render-utils.js';
 
 checkAuth();
 
 const logoutButton = document.getElementById('logout');
+const formEl = document.querySelector('form');
+const listEl = document.querySelector('.shopping-list');
+const deleteButton = document.getElementById('delete-list');
+
 
 logoutButton.addEventListener('click', () => {
     logout();
 });
+
+formEl.addEventListener('submit', async (e) => {
+    e.preventDefault();
+
+    const data = new FormData(formEl);
+
+    await createListItem({
+        amount: data.get('amount'),
+        item_name: data.get('item'),
+        is_bought: false,
```

Comment:
`is_bought: false` totally works, so not necessarily saying change it. But, if in supabase you set the default to that column to false you won't have to do it here. It lets us as devs be more lazy 😎